webidl/ecmascript-binding/es-exceptions/DOMException-is-error.any.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
```

Comment:
Consider adding this:

```suggestion
// META: global=window,dedicatedworker,shadowrealm

'use strict';
```

(see #49159)